### PR TITLE
[DOCFIX] Update Glossary Page & minor changes

### PR DESCRIPTION
**Pull request by ccmao1130**
### What changes are proposed in this pull request?

Added more terms to glossary page

### Why are the changes needed?

update terms list & definitions

### Does this PR introduce any user facing changes?

yes

### CODE REVIEW

File: docs/en/Overview.md [hunk 1]
```diff
@@ -7,12 +7,22 @@ Welcome to Alluxio Documentation! You will find resources regarding deploying Al
...
-<iframe width="560" height="315" src="https://www.youtube.com/embed/5YQvvznT5cI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
-
 Check out our [Quick Start Guide]({{ '/en/Get-Started.html' | relativize_url }}) for step-by-step guidance on how to download Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5YQvvznT5cI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
 Follow [Install Alluxio Cluster with HA]({{ '/en/deploy/Install-Alluxio-Cluster-with-HA.html' | relativize_url }}) to deploy an Alluxio cluster with High Availability (HA) by running multiple Alluxio master processes on different nodes in the system.
 
+## Kubernetes
+See [Install Alluxio on Kubernetes]({{ '/en/kubernetes/Install-Alluxio-On-Kubernetes.html' | relativize_url }}) on how to install Alluxio (Dora) on Kubernetes via
```
Comment (Xenorith) [inline docs/en/Overview.md:17]:
```suggestion
See [Install Alluxio on Kubernetes]({{ '/en/kubernetes/Install-Alluxio-On-Kubernetes.html' | relativize_url }}) on how to install Alluxio on Kubernetes via 
```

File: docs/en/Overview.md [hunk 2]
```diff
@@ -7,12 +7,22 @@ Welcome to Alluxio Documentation! You will find resources regarding deploying Al
...
-
 Check out our [Quick Start Guide]({{ '/en/Get-Started.html' | relativize_url }}) for step-by-step guidance on how to download Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5YQvvznT5cI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
 Follow [Install Alluxio Cluster with HA]({{ '/en/deploy/Install-Alluxio-Cluster-with-HA.html' | relativize_url }}) to deploy an Alluxio cluster with High Availability (HA) by running multiple Alluxio master processes on different nodes in the system.
 
+## Kubernetes
+See [Install Alluxio on Kubernetes]({{ '/en/kubernetes/Install-Alluxio-On-Kubernetes.html' | relativize_url }}) on how to install Alluxio (Dora) on Kubernetes via 
+[Helm](https://helm.sh/), a kubernetes package manager, and [Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/), a kubernetes extension for managing applications.
```
Comment (Xenorith) [inline docs/en/Overview.md:18]:
```suggestion
[Helm](https://helm.sh/), a Kubernetes package manager, and [Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/), a Kubernetes extension for managing applications.
```

File: docs/en/introduction/Glossary.md [hunk 3]
```diff
@@ -3,21 +3,64 @@ layout: global
...
+
+Consistent Hashing
+: Consistent hashing is a technique used in distributed systems to efficiently distribute data across multiple nodes in a way that minimizes data movement and disruption when nodes are added or removed from the system. [See Consistent Hashing in Dora Cache]({{ '/en/core-services/Data-Caching.html#load-balancing-between-dora-nodes' | relativize_url }})
+
+Data Caching
+: Alluxio provides an in-memory data caching mechanism to accelerate data access. Frequently accessed data is cached in memory, reducing the need for repetitive disk I/O and improving overall application performance. [Learn more]({{ '/en/core-services/Data-Caching.html' | relativize_url }})
+
+Fault Tolerance
+: Alluxio incorporates fault tolerance mechanisms to ensure dat availability and reliability. It replicates data across different notes in the cluster, providing resilience against node failures. [See Fault Tolerance in Dora Cache]({{ '/en/core-services/Data-Caching.html#fault-tolerance-and-client-side-ufs-fallback' | relativize_url }})
```
Comment (Xenorith) [inline docs/en/introduction/Glossary.md:19]:
```suggestion
: Alluxio incorporates fault tolerance mechanisms to ensure data availability and reliability. It replicates data across different notes in the cluster, providing resilience against node failures. [See Fault Tolerance in Dora Cache]({{ '/en/core-services/Data-Caching.html#fault-tolerance-and-client-side-ufs-fallback' | relativize_url }})
```

**Review by Xenorith — CHANGES_REQUESTED** [review]

**Review by Xenorith — APPROVED** [review]

**Review by Zijian-Zhu — APPROVED** [review]

**Comment by apc999** [comment]
@ccmao1130 is this PR ready to merge?

**Comment by Zijian-Zhu** [comment]
alluxio-bot, merge this please

**Pull request merged**